Clinical-Genomics/scout
# switching buttons

This PR swhitces place on the buttns "Next page" and "First page" as described in this [issue](https://github.com/Clinical-Genomics/scout/issues/1367)
It also adds the functionality of next page to the cancer view

**How to test**:
1. Install on clinical db by running:
`bash servers/resources/clinical-db.scilifelab.se/update-scout-stage.sh new_switch_buttons`
2. Go to : https://scout-stage.scilifelab.se 
3. Look at the different variant views. The Next page button should be last. and the cancer view should have the new buttons at the bottom of the page.


**Review:**
